rpc/legacyrpc/methods.go

```diff
@@ -2160,6 +2161,116 @@ func purchaseTicket(s *Server, icmd interface{}) (interface{}, error) {
 	return hashStrs, err
 }
 
+// createUnsignedTicket creates unsigned ticket transactions.
+// If the ticket could not be purchased because there are not enough eligible
+// funds, an error will be returned.
+func createUnsignedTicket(s *Server, icmd interface{}) (interface{}, error) {
```

Comment:
Don't duplicate this code.  Modify purchaseTickets to allow specifying an unsigned request instead.